client/my-sites/backup/backup-contents-page/file-browser/file-info-card.tsx

```diff
@@ -60,8 +60,8 @@ function FileInfoCard( {
 
 	// Dispatch an error notice if the download could not be prepared
 	const handlePrepareDownloadError = useCallback( () => {
-		dispatch( onPreparingDownloadError() );
-	}, [ dispatch ] );
+		notices.showError( __( 'There was an error preparing your download. Please, try again.' ) );
```

Comment:
I understand the actual message has been copied from the original code, but maybe we could consider the following adjustment:

---

Looking at the other existing messages in the Calypso UI, we mostly skip comma after `Please`, i.e.:

```suggestion
		notices.showError( __( 'There was an error preparing your download. Please try again.' ) );
```